app/views/guidance/send_by_roro.scala.html

```diff
@@ -0,0 +1,122 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.helpers.Title
+@import views.html.components.gds._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(
+    govukLayout: gdsMainTemplate,
+    pageTitle: pageTitle,
+    paragraphBody: paragraphBody
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@govukLayout(title = Title("guidance.roro.title"), useCustomContentWidth = true) {
+
+    <span class="govuk-caption-xl">Guidance</span>
```

Comment:
@sectionHeader("Guidance")

Reply:
Changed this to use the component, will update this on the error codes page as well.